crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -374,7 +365,7 @@ fn extract_descriptions(wasm_file: &Path) -> anyhow::Result<ModuleDef> {
             }
         },
     )?;
-    linker.func_wrap("spacetime", "_buffer_alloc", WasmCtx::buffer_alloc)?;
+    linker.func_wrap("spacetime_6.0", "_buffer_alloc", WasmCtx::buffer_alloc)?;
```

Comment:
Nit: extract the constant somewhere at the top of the file?

Reply:
oh maybe I'll just use the constant from spacetimedb-lib